pepper-apis/dss-core/src/main/java/org/broadinstitute/ddp/json/AgeUpResponse.java

```diff
@@ -0,0 +1,16 @@
+package org.broadinstitute.ddp.json;
+
+import com.google.gson.annotations.SerializedName;
+
+public class AgeUpResponse {
```

Comment:
Let's add a Lombok `@Value` annotation to this class so we'll have the getters generated.